quickstart/application/configs/application.ini

```diff
@@ -0,0 +1,198 @@
+[production]
```

Comment:
how about changing this to : application.ini.example, then only commit this file. in your local, you can make a copy of it and customize it